app/assets/stylesheets/admin/checkout_products.css

```diff
@@ -30,6 +30,28 @@ table.table.table-linked > tbody > tr > td > a:hover {
   background: transparent;
 }
 
+.update-side-bar {
```

Comment:
The layout is unusable when on half screen. Both fullscreen and mobile work fine

Reply:
Fixed in new d9ebbc9